lib/index.js

```diff
@@ -54,6 +59,15 @@ function downloadFromInfoCallback(stream, info, options) {
   req.on('response', function(res) {
     stream.emit('response', res);
     if (res.statusCode !== 200) {
+      if(res.statusCode == 302) {
+        //Redirection header//
+        console.log("[Bugfixtest] Got 302 header; trying fix");
+        var location = res.rawHeaders.indexOf('Location')+1;
```

Comment:
why not use `res.headers`?
https://nodejs.org/api/http.html#http_message_headers